driver-core/src/main/com/mongodb/internal/capi/MongoCryptHelper.java

```diff
@@ -72,9 +78,31 @@ public static MongoCryptOptions createMongoCryptOptions(final Map<String, Map<St
             spawnArgs.add("--idleShutdownTimeoutSecs");
             spawnArgs.add("60");
         }
+        if (!spawnArgs.contains("--logpath")) {
+            spawnArgs.add("--logpath");
+            spawnArgs.add(System.getProperty("os.name").startsWith("Windows") ? "NUL" : "/dev/null");
```

Comment:
Borrowed this idea from Java 8 Redirect.DISCARD

Reply:
Nice - didn't know about NUL for windows.